### PR TITLE
Translation and fix image btc 101

### PR DESCRIPTION
**Pull request by Rogzy-DB**
Ok to merge. massive changes so double check: 

- creation of new language separatio for image: 

Assets/Concept_Language/chapter1

batch changes of image name for translated language

fix of image of chapter 19
move all image to png format

Figma files of the image translation: https://www.figma.com/file/5FRdXr0SHEashxaHWFh0bQ/Untitled?type=design&node-id=24%3A208234&mode=design&t=ILhFOwlQYOppcxhU-1

Translation done via ChatGPT. 

add of the poster for EN FR and ES. not poster modification for IT and DE

### CODE REVIEW

**Comment by Asi0Flammeus** [comment]
I do not agree with the path systematics. Below you can find reasons why by expliciting the context and the problem we want to solve, then describing your solution, and explaining why it makes repo maintenance _and_ content translation harder. I will finish with a proposal of my own to solve the initial problem. 

If you agree I will push the changes to your `Translation-and-fix-image-BTC-101` branch. 

(sorry in advance for the long comment) 

## TL;DR

- we want to translate images with linguistic content 
- we need a consistent naming convention to facilitate the repo maintenance and translation in other languages
  - avoid useless image duplication
  - make image path link descriptive and easy to automatically modify when translating the markdown file  
- rogzy's naming convention proposal has some shortcomings: 
  - would not be consistent with non-latin-script language (like arabic or japanese) 
  - chapter folder and image file name is based on their ordering in the markdown file which descrease the flexibility for later modification (or at least increase the propability of future non-intentional error)
  - the `Concept_Language` folder leads to a duplication of image files with non-linguistic content 
  - here's examples of image file:
    - `assets/Concept_FR/chapitre1/image1.png` 
    - `assets/Concept_EN/chapter1/image1.png` 
- asi0's naming convention proposal is an attempt to organise with a low-time preference  
  - chapter folder and file name must have a descriptive name following the `snake_case` with respectively 3 and 4 words max
  - image file name should end with the language code, and for non-linguistic content the language code `xx` is used to avoid duplication
  - here's an 3 examples of path for image files
    - `assets/bitcoin_network_topology/kind_of_topology_networks_fr.png`
    - `assets/bitcoin_network_topology/kind_of_topology_networks_en.png`
    - `assets/consensus_rules/screenshot_bitcoin_core_code_xx.png`
  - the drawback is that is demand a high initial effort to give descriptive names for chapter folder and image file


## Context

Most of the content that we host in this repo has images in their body. These images are stored in the `/asset` and we use the markdown syntax `this-is-an-image` to point to a specific image in the `/asset` folder. 

Up to now, everythings worked fine despite the lack of a precise and systematic hierarchy for the inner of the `/asset` folder. However it is not possible anymore once we want to store translated version of schemes, drawing, infography or any other kind of image where a translation would be required. 

But we have to keep in mind that: 
1. we **can not duplicate images that are not necessary to duplicate** (i.e. with no linguistic content) in order to limit the weight of the repo.
2. we need a **naming convention for path and image files** that is easy to understand (i.e. know what's it is about and where it is located in the content) and that is easy to automatically modify in the associated markdown file when the translation in another language is applied.

## Rogzy's proposal 

You have started a major project that is the translation of images with linguistic content that is relevant to translate. In particular, you have already translated all these files for the course `btc101` in french, german, spanish, and italian. 

The path naming convention that you used is `assets/Concept_Language/chapter1/image1.png` where the language is replace with the code language the subfolder `chapter1` is also translated in language in the upper folder (e.g. `assets/Concept_DE/kapitel1/image1.png`). Moreover you have duplicated all the images files in this subfolder, even those with no linguistic content. 

Therefore inside the german btc101 markdown file it will look like this (here I took as examples this english and french version): 

```
## Consensus Rules define Bitcoin

...
[image](assets/Concept_EN/chapter1/image1.png) (non-linguistic content)
...
[image](assets/Concept_EN/chapter1/image2.png)


## Bitcoin Network Topology ensures decentralization

...
[image](assets/Concept_EN/chapter2/image1.png) 
...
[image](assets/Concept_EN/chapter2/image2.png)


```

```
## Les Règles de Consensus définisse Bitcoin

...
[image](assets/Concept_FR/chapitre1/image1.png) (non-linguistic content)
...
[image](assets/Concept_FR/chapitre1/image2.png)


## La topologie du réseau Bitcoin assure sa décentralisation

...
[image](assets/Concept_FR/chapitre2/image1.png)
...
[image](assets/Concept_FR/chapitre2/image2.png)


```

This proposed solution can get quickly problematic for the following reasons: 
1. translating a part of the path in the defined language would be infeasible for non-Latin script language like japanese, or arabic. In those case, we will have be non-consistent and keep the subfolder name `chapter1`. 
2. because of non-linguistic image duplication the asset folder will significantly grows that the translation of the image is done. This is not acceptable as the biggest part of the repo's weigth is already the images files. 
3. naming image file by simply indexing their order in the markdown is not optimal because just by looking at the name file we have no idea of the image content and if one wants to re-order in the markdown file, he has to also modify the name of all files in between. This points also stand for the naming convention of the subfolder related to the chapter. 
4. Defining the initial subfolder `Concenpt_Language` seems an unnecessary subfolder as the language of the file can be directly defined in the file name. 


## asi0's proposal 

Based on the need to avoid useless duplication and maintain effective hierarchy for translated images I propose a naming convention that enforced a consistent hierarchy based on these rules:
1. descriptive names for chapter folder and image file names using the `snake_case` as such:
   - `assets/three_word_max/` to describe the associated chapter -- allowing for re-ordering the chapters without the need to change the path of all associated images. 
   - `/up_to_four_words_{code_language}.png` to describe image content -- allowing for re-ordering the chapters without the need to change the path of all associated images
   - the code_language for non-linguistic content is `xx` as it is often used in multi-languages system that use ISO 639-1 standard
   - the description for chapter and image is always written in english
2. every translated _and_ non-linguistic image files in a defined chapter share the same folder -- allowing a reduction of the path length and the number of changes once we need to translate of the markdown file in other language. 

Using the previous example and my proposed convention, it renders like this 

```
## Consensus Rules define Bitcoin

...
[image](assets/consensus_rules/screenshot_bitcoin_core_code_xx.png) (non-linguistic content)
...
[image](assets/consensus_rules/consensus_rules_infography_en.png)


## Bitcoin Network Topology ensures decentralization

...
[image](assets/bitcoin_network_topology/kind_of_topology_networks_en.png)
...
[image](assets/bitcoin_network_topology/roles_of_bitcoin_nodes_en.png)


```

```
## Les Règles de Consensus définisse Bitcoin

...
[image](assets/consensus_rules/screenshot_bitcoin_core_code_xx.png) (non-linguistic content)
...
[image](assets/consensus_rules/consensus_rules_infography_fr.png)


## La topologie du réseau Bitcoin assure sa décentralisation

...
[image](assets/bitcoin_network_topology/kind_of_topology_networks_fr.png)
...
[image](assets/bitcoin_network_topology/roles_of_bitcoin_nodes_fr.png)


```

The drawback of this naming convention is that is initial effort to be adopted, as it requires to go through all the material and give descriptive names to _each_ chapter folder and image files. 
However I'm convinced that this effort is worthwhile as it is aligned with a low-time preference to lower the future load of repo maintainance and content translation.
